### ISSUE TITLE
error: bad symbolic reference w/ sbt 0.13.8

### ISSUE THREAD

**Issue by oschrenk** [rtimush/sbt-updates#81]
I have a client project using sbt 0.13.8 with which I get

```
[info] Loading global plugins from /Users/oliver/.sbt/0.13/plugins
error: bad symbolic reference. A signature in Implicits.class refers to term Remove
in package sbt which is not available.
It may be completely missing from the current classpath, or the version on
the classpath might be incompatible with the version used when compiling Implicits.class.
error: bad symbolic reference. A signature in Implicits.class refers to type Value
in value sbt.Remove which is not available.
It may be completely missing from the current classpath, or the version on
the classpath might be incompatible with the version used when compiling Implicits.class.
[error] Type error in expression
```

when the sbt-updates plugin is enabled

```
$ cd ~/.sbt/0.13/plugins
$ cat sbt-updates.sbt
addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.1")
```

I noticed that the requirements say
> SBT 0.13.5 and later. SBT 1.0 is supported since version 0.3.1.
> Note: use version 0.1.0 for SBT 0.11.x, version 0.1.6 for SBT 0.12.x, version 0.1.7 for SBT 0.13.0-0.13.2.

It basically says that you can't use sbt 0.13.x with 3 <= x < 15. Which of course then means that the requirements state that the version 0.13.8 is not supported and I will try to push the client to upgrade to a later version, but...

Out of curiosity, can you tell me what the problem with these versions of sbt is?

**Comment by rtimush** [comment]
Thank you for reporting. The intention was to support `0.13.8` but the compatibility was not tested and therefore at some point it got broken. As a part of a fix for another SBT compatibility issue (#79) I introduced compatibility testing but there are still some issues with older SBTs.
Now I need to go through the old versions and fix everything, shouldn't be too difficult.
In the meanwhile, you can try older `sbt-updates` versions, although pushing your client to update SBT seems to be a good move anyway.

**Comment by oschrenk** [comment]
With sbt 1.0 being very usable now and with 0.13.8 so far in the past, I'm closing this.
